Resources/Prototypes/Entities/Objects/Fun/Tabletop/parchis.yml

```diff
@@ -0,0 +1,74 @@
+# Parchís board item (normal in game item you can hold in your hand)
+- type: entity
+  parent: BaseItem
+  id: ParchisBoard
+  name: parchís board
```

Comment:
Parchís should probably be capitalized.

Reply:
that's against the naming guidelines

Reply:
Hmm I assumed there would be an exception for names which need capitalization

Reply:
I mean yeah that's a thing but, I don't think parchís needs to be capitalized? no clue now honestly

Reply:
Wikipedia definitely does it: https://en.wikipedia.org/wiki/Parch%C3%ADs

For example "Since it uses dice, Parchís is not usually [...]" and "As in Parchis, the aim is simple [...]".